### DISCUSSION TITLE
[Feature] Ability to hide the toolbar in fullscreen

### DISCUSSION THREAD

**Discussion by dagstuan** [immich-app/immich#10083] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice with a way of hiding the toolbar when viewing photos full screen. Currently the toolbar can obfuscate parts of the image being viewed, and causes images to have a dark gradient at the top. It would be nice with a way to hide it, such as a keyboard shortcut or button. See screenshot.

<img width="1512" alt="image" src="https://github.com/immich-app/immich/assets/397559/1691a617-8c0e-43c6-8f36-80f1ef059cbc">

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by sisyphosloughs** [comment]
I came across Immich and find the app almost perfect!

However, the display of the controls is so distracting for me and the people I share my photos with that I will continue to use [Photoview](https://photoview.github.io/), even though the app is significantly slower. Take a look at how this is implemented in Photoview. After a few seconds, the controls disappear and reappear only when the mouse pointer moves over the webpage again. This also works in the mobile view, by the way. I would really like to see this functionality here as well.

**Comment by rome2** [comment]
On any event that some one else stumbles upon this.  I removed the gradient with this custom CSS:

`
.bg-linear-to-b {
      --tw-gradient-from: transparent;
}

.from-black\/40 {
      --tw-gradient-from: transparent;
}
`

Sadly this also removes the mouse over darkening when in thumbnail view but I can live with that. I don't know guts of the immich source code so I have no idea if this has any other side effects. But up to now it looks fine.

I really love immich and I agree that it is almost perfect. But this 'feature' just looks horrible on carefully composed images, especially when viewed on a big screen. It would be nice if the buttons could vanish automatically but they are not as disturbing as a ugly gradient overlay on every single picture.
